### PR TITLE
Added checkpoint to confirm no error msg shown

### PR DESCRIPTION
**Pull request by iccaszhulili**
Make sure no unexpected error shown in log when
connecting qemu:///system with non-root user

Signed-off-by: Lili Zhu <lizhu@redhat.com>

Bug: 1945501

# avocado run --vt-type libvirt virsh.non_root_polkit.positive_testing.virsh_connect_non_root
No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : 448e88f0ed6ed6f45e875f2fc491c8dbef6fe487
JOB LOG    : /root/avocado/job-results/job-2022-05-07T03.17-448e88f/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.non_root_polkit.positive_testing.virsh_connect_non_root: PASS (21.59 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 23.13 s

Also Check CI job libvirt-RHEL-9.1-provision-x86_64-function-remote_access/8/

### CODE REVIEW

**Comment by iccaszhulili** [comment]
@dzhengfy Please help to review. Thanks

File: libvirt/tests/src/remote_access/virsh_non_root_polkit.py [hunk 1]
```diff
@@ -120,6 +124,33 @@ def create_non_root_user(params, test):
         test.error("Unexpected error: {}".format(e))
 
 
+def get_log():
```
Comment (chunfuwen) [inline libvirt/tests/src/remote_access/virsh_non_root_polkit.py:127]: get_log change to get_tailed_message_log?

Comment (iccaszhulili) [reply libvirt/tests/src/remote_access/virsh_non_root_polkit.py]: @chunfuwen Updated. Thanks

**Review by chunfuwen — APPROVED** [review]
LGTM

**Review by cliping — APPROVED** [review]
LGTM

File: libvirt/tests/src/remote_access/virsh_non_root_polkit.py [hunk 2]
```diff
@@ -120,6 +124,33 @@ def create_non_root_user(params, test):
...
+    :returns: the appended data tailed from /var/log/messages
+    """
+    tailed_log_file = os.path.join(data_dir.get_tmp_dir(), 'tail_log')
+    tailed_messages = aexpect.Tail(command='tail -f /var/log/messages',
+                                   output_func=utils_misc.log_line,
+                                   output_params=(tailed_log_file))
+    return tailed_messages
+
+
+def check_tailed_messages(params, output, test):
```
Comment (dzhengfy) [inline libvirt/tests/src/remote_access/virsh_non_root_polkit.py:140]: No need to define this function.

File: libvirt/tests/src/remote_access/virsh_non_root_polkit.py [hunk 3]
```diff
@@ -128,11 +159,15 @@ def run(test, params, env):
...
     create_non_root_user(params, test)
+    tailed_messages = get_tailed_message_log()
     try:
         if ssh_connection:
             ssh_access(params, test)
         else:
             local_access(params, test)
+        output = tailed_messages.get_output()
+        logging.debug("Get messages in /var/log/messages: %s" % output)
+        check_tailed_messages(params, output, test)
```
Comment (dzhengfy) [inline libvirt/tests/src/remote_access/virsh_non_root_polkit.py:170]: Could you help consider to reuse this function virttest/utils_test/libvirt.py `check_logfile('error :', '/var/log/messages', str_in_log=False)`

**Review by dzhengfy — COMMENTED** [review]
Others LGTM

File: libvirt/tests/src/remote_access/virsh_non_root_polkit.py (hunk 3 above)
Comment (iccaszhulili) [reply libvirt/tests/src/remote_access/virsh_non_root_polkit.py]: @dzhengfy Updated. Thanks

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**